Add fix in the poll element (#166)
bsi-software/bsi-cx-design-standard-library-web: CHANGELOG.md

* Add dark mode support for general layout and all elements
* Remove empty `ul`-tag from form summary validation 
* Fix the bootstrap date icon visualization

> * Fix poll element dark mode
> * Fix poll element tabnavigation

Reviewer: ```suggestion
* Update dark mode and accessibility for poll element
```
